tests/term1.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+
+# system utilities stubs
+
+# UUT
+. ../share/pot/term.sh
+
+. ../share/pot/common.sh
+# common stubs
+. common-stub.sh
+
+# app specific stubs
+
+pot-cmd()
+{
+	__monitor POTCMD "$@"
+	if [ "$POTCMD_SHOULD_START_POT" = "yes" ]; then
+		_pname="test-pot-run"
+	fi
+}
+
+
+_term()
+{
+	__monitor TERM "$@"
+}
+
+term-help()
+{
+	__monitor HELP "$@"
+}
+
+test_pot_term_001()
+{
+	pot-term
+	assertEquals "Exit rc" "1" "$?"
+	assertEquals "Help calls" "1" "$HELP_CALLS"
+	assertEquals "Error calls" "1" "$ERROR_CALLS"
+	assertEquals "_is_pot_running calls" "0" "$ISPOTRUN_CALLS"
+	assertEquals "_term calls" "0" "$TERM_CALLS"
+	assertEquals "pot-cmd calls" "0" "$POTCMD_CALLS"
+
+	setUp
+	pot-term -b bb
+	assertEquals "Exit rc" "1" "$?"
+	assertEquals "Help calls" "0" "$HELP_CALLS"
+	assertEquals "Error calls" "1" "$ERROR_CALLS"
+	assertEquals "_is_pot_running calls" "1" "$ISPOTRUN_CALLS"
+	assertEquals "_term calls" "0" "$TERM_CALLS"
+	assertEquals "pot-cmd calls" "0" "$POTCMD_CALLS"
+
+	setUp
+	pot-term -h
+	assertEquals "Exit rc" "0" "$?"
+	assertEquals "Help calls" "1" "$HELP_CALLS"
+	assertEquals "Error calls" "0" "$ERROR_CALLS"
+	assertEquals "_is_pot_running calls" "0" "$ISPOTRUN_CALLS"
+	assertEquals "_term calls" "0" "$TERM_CALLS"
+	assertEquals "pot-cmd calls" "0" "$POTCMD_CALLS"
+}
+
+test_pot_term_020()
+{
+	pot-term test-pot-run
+	assertEquals "Exit rc" "0" "$?"
+	assertEquals "Help calls" "0" "$HELP_CALLS"
+	assertEquals "Error calls" "0" "$ERROR_CALLS"
+	assertEquals "_is_pot_running calls" "1" "$ISPOTRUN_CALLS"
+	assertEquals "_term calls" "1" "$TERM_CALLS"
+	assertEquals "pot-cmd calls" "0" "$POTCMD_CALLS"
+}
+
+test_pot_term_030()
```

Comment:
I would split this test in 3 different tests

Reply:
Done.